geolib/models/dsettlement/internal.py

```diff
@@ -1014,10 +1014,13 @@ class ResidualSettlements(DSerieOldTableStructure):
     # TODO LIst[Dict[str, float]] but can be empty which now gives a validation error
     residualsettlements: List[Dict[str, float]]
 
+class CalculationSettings(DSeriesStructure):
+    is_secondary_swelling_used: bool = False
```

Comment:
Add a comment explaining that this option is in Evaluation and can be used only with a development version of DSettlement